src/patternfly/components/Toolbar/examples/Toolbar.md

```
@@ -557,6 +557,246 @@ Several components in the following examples do not include functional and/or ac
+    {{/toolbar-content-section}}
+  {{/toolbar-content}}
+{{/toolbar}}
+  </div>
+      <div id="ws-core-spacer-container">290px spacer for demonstration purposes</div>
+
+  </div>
+  </div>
+</div>
+```
+
+### Toolbar with container query support for hidden/visible items and groups
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)